src/test/java/core/basesyntax/strategy/impl/ReturnOperationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnOperationHandlerTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private FruitTransaction fruitTransaction;
+    private ReturnOperationHandler returnOperationHandler = new ReturnOperationHandler();

```

Comment:
Use @BeforeAll to create an instance of ReturnOperationHandler and  FruitTransaction